libs/status/type-defs/status-payloads/error-info.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @todo
```

Comment:
This `@todo` is empty, I suppose that isn't intentional

Reply:
Nope, that was indeed unintentional, wanted to add some docs about what the `ErrorInfo` is useful for